core/kazoo_services/src/kazoo_services_maintenance.erl

```diff
@@ -207,7 +211,11 @@ js_quantify_map() ->
       ,kazoo_number_manager_maintenance:generate_js_classifiers(FunMatchBlock),
        "            break;"
        "        case 'qubicle_queue':"
-       "            emit(['qubicle', 'queues'], 1);"
+       "            if (doc.pvt_prod_level == 'pro') {"
```

Comment:
@mark2600 same with above, this could be simplified to emit(['qubicle', doc.pvt_qubicle_offering || 'basic' + '_queue'], 1);